src/bidiMapper/domains/network/NetworkProcessor.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type {Network, EmptyResult} from '../../../protocol/protocol.js';
+
+import {NetworkStorage} from './NetworkStorage.js';
+
+export class NetworkProcessor {
+  #networkStorage: NetworkStorage = new NetworkStorage();
+
+  process_network_addIntercept(
```

Comment:
Drop the process_network_ we are in the networkProccessor its doubling the text. Also I already did that for the other Processors so we should be consistent.